src/pages/home/report/ReportActionCompose.js

```diff
@@ -797,13 +797,13 @@ class ReportActionCompose extends React.Component {
             this.debouncedUpdateFrequentlyUsedEmojis();
         }
 
-        this.setState((prevState) => {
+        this.setState(() => {
             const newState = {
                 isCommentEmpty: !!newComment.match(/^(\s)*$/),
                 value: newComment,
             };
             if (comment !== newComment) {
-                const remainder = prevState.value.slice(prevState.selection.end).length;
+                const remainder = ComposerUtils.getCommonSuffixLength(comment, newComment);
```

Comment:
Coming from https://github.com/Expensify/App/issues/27195, this has caused a small inconsistency between the composer when sending a message and when editing it. 
We should have used the same calculation for the remainder (`getCommonSuffixLength`) in `ReportActionItemMessageEdit` too.